boards/arm/frdm_k64f/Kconfig.defconfig

```diff
@@ -43,20 +43,16 @@ config I2C_0
 	default y
 	depends on I2C
 
+config ADC_0
+	default y
+
 config ADC_1
 	default y
 	depends on ADC
 
 config TEMP_KINETIS
 	default y if ADC_1
 	depends on SENSOR
-if DAC
-
-config DAC_0
-	default y
-
-endif # DAC
```

Comment:
rebasing mistake?

Reply:
Not sure where that came from. Here it looks like you commented on an outdated change set.